src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,24 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    public static final int RESULT_MAX_LENGTH = 5;
+
+    public static String getUniqueCharacters(String fileName) throws IOException {
+        Set<String> treeSet;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            treeSet = bufferedReader.lines()
+                    .flatMapToInt(s -> s.toLowerCase().chars())
+                    .mapToObj(c -> (char) c)
+                    .filter(Character::isLetter)
+                    .map(String::valueOf)
+                    .sorted()
+                    .collect(Collectors.toCollection(TreeSet::new));
+        } catch (IOException e) {
+            throw new FileNotFoundException("File not found");
+        }
```

Comment:
you will wrap any IOException  to FileNotFoundException??let's just leave only throws IOException in this case